src/qt/pivx/forms/createproposaldialog.ui

```diff
@@ -1270,15 +1273,40 @@
        </widget>
       </item>
       <item>
-       <layout class="QHBoxLayout" name="horizontalLayout1" stretch="1,2,2">
+       <layout class="QHBoxLayout" name="horizontalLayout1" stretch="0,0,0,2">
+        <item>
+         <widget class="QPushButton" name="btnGenAddr">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>50</height>
+           </size>
+          </property>
+          <property name="maximumSize">
+           <size>
+            <width>200</width>
+            <height>16777215</height>
+           </size>
+          </property>
+          <property name="focusPolicy">
+           <enum>Qt::NoFocus</enum>
+          </property>
+          <property name="text">
+           <string>GENERATE ADDRESS</string>
```

Comment:
nit: Since this text is being added programmatically in the `.cpp` file, there is no need to also define it here.
can instead use `<string notr="true">N/A</string>`